index.js

```diff
@@ -1,46 +1,113 @@
 const cheerio = require('cheerio');
 
-const translations = {
+//german translations for attrs
+const attr_translations = {
     'Latein': 'latin',
     'Typ': 'type',
     'Flexionsart': 'flexion_type',
     'Form': 'form',
     'Deutsch': 'german',
-    'Geschlect': 'gender'
+    'Geschlecht': 'gender'
 };
 
+//word information
 module.exports.getWordInformation = async function (word) {
+    //send http request
     let res = await centra(`https://www.frag-caesar.de/lateinwoerterbuch/${word}-uebersetzung.html`).send();
-    if (res.body.toString().includes(`Ihr Suchwort <strong><span class="textmarker">${word}</span></strong> entspricht`)) res = await centra(`https://www.frag-caesar.de/lateinwoerterbuch/${word}-uebersetzung-1.html`).send();
+
+    //when more results exist
+    if (res.body.toString().includes(`Ihr Suchwort <strong><span class="textmarker">${word}</span></strong> entspricht`))
+        //redirect to first result
+        res = await centra(`https://www.frag-caesar.de/lateinwoerterbuch/${word}-uebersetzung-1.html`).send();
+
+    //load content
     const $ = await cheerio.load(res.body.toString());
+
+    //put all table rows in array
     const tableElements = $('.table-responsive table tr').toArray();
+
+    //init counter and content for loop
     let i = 0;
     const content = {};
-    tableElements.forEach(element => {
+
+    //for each tr
+    tableElements.forEach(tr => {
+        //increment counter
         i++;
+
+        //add array for current table row
         if (!content[i]) content[i] = [];
-        element.children.forEach(children => {
+
+        //for each td
+        tr.children.forEach(td => {
+            //init child content
             let childContent = '';
-            if (children.children) children.children.forEach(c => {
+
+            //for each td child as c (different translation of word)
+            if (td.children) td.children.forEach(c => {
+                //add c to content string
                 if (c.data) childContent = childContent + `${c.data}|`;
             });
+
+            //push td content string to content array
             content[i].push(childContent);
         });
     });
 
+    //init result
     const returnForm = {};
 
+    //return if content does not exist
     if (!content[1]) return null;
 
+    //init counter
     let y = 0;
+
+    //for each td content string from first tr as c
     content[1].forEach(c => {
-        if (content[3][y]) returnForm[translations[c.split('|').join('')] ? translations[c.split('|').join('')] : c.split('|').join('')] = content[3][y].substring(0, content[3][y].length - 1);
-        else returnForm[translations[c.split('|').join('')] ? translations[c.split('|').join('')] : c.split('|').join('')] = content[2][y].substring(0, content[2][y].length - 1);
+        //get keys
+        var key_returnForm = attr_translations[c.split('|').join('')] ? attr_translations[c.split('|').join('')] : c.split('|').join('');
```

Comment:
`var` should not be used anymore, please use `let`/`const` depending on the use-case. [Learn more](https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994).


`c.split('|').join('')` could also be replaced by `.replaceAll('|', '')`, but this is more of a suggestion than something that acutally needs to be changed